### PR TITLE
[Issue-15]Adding instance locale support

### PR DESCRIPTION
**Pull request by Vishwaas**
@cibernox Ready for review with regards to [discussed issue](https://github.com/cibernox/ember-power-datepicker/issues/15)

### CODE REVIEW

File: tests/dummy/config/environment.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ module.exports = function(environment) {
     environment: environment,
     rootURL: '/',
     locationType: 'auto',
+    moment: {
+      includeLocales: ['ru']
```
Comment (cibernox) [inline tests/dummy/config/environment.js]: Change this to `includeLocales: true` and then you can remove the changes you made in _tests/dummy/app/controllers/public-pages/index.js_ and tests should be green.

Comment (Vishwaas) [inline tests/dummy/config/environment.js]: Was not sure including all locales just for demo purpose was needed, sure will make the change

**Comment by Vishwaas** [comment]
@cibernox  Changes made

**Comment by cibernox** [comment]
Thanks. I'll publish a new version later

**Comment by Vishwaas** [comment]
Great, thanks

**Pull request merged**